code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -599,6 +604,7 @@
 Returns the path corresponding to the corresponding organ
 It'll return null if the organ doesn't correspond, so include null checks when using this!
 */
+//Fethas Todo:Do i need to redo this?
```

Comment:
I use this in MMI code to get the organ corresponding to the organ tag for a species, in particular for transferring a human's brain to an MMI. This just returns the path, so as things stand right now it should work fine, given that you haven't modified the species' has_organ list. You might want to swap out the calls of this in MMI.dm with get_int_organ, though